app/gui2/src/components/ComponentBrowser/input.ts

```diff
@@ -81,58 +80,72 @@ export function useComponentBrowserInput(
   suggestionDb: SuggestionDb = useSuggestionDbStore().entries,
   ai: { query(query: string, sourcePort: string): Promise<Result<string>> } = useAI(),
 ) {
-  const code = ref('')
+  const text = ref('')
   const cbUsage = ref<Usage>()
   const anyChange = ref(false)
   const selection = ref({ start: 0, end: 0 })
-  const ast = computed(() => RawAstExtended.parse(code.value))
+  const ast = computed(() => RawAstExtended.parse(text.value))
   const imports = ref<RequiredImport[]>([])
   const processingAIPrompt = ref(false)
   const toastError = useToast.error()
+  const sourceNode = ref<string>()
   const firstAppliedReturnType = ref<Typename>()
 
-  // Code Model to being edited externally (by user).
+  // Text Model to being edited externally (by user).
   //
   // Some user actions (like typing operator right after input) may handled differently than
   // internal changes (like applying suggestion).
-  const codeModel = computed({
-    get: () => code.value,
-    set: (newValue) => {
-      if (newValue != code.value) {
-        // When user, right after opening CB with source node types operator, we should
-        // re-initialize input with it instead of dot at the end.
```

Comment:
Could we keep this comment? It may be helpful, as what we do here is quite unusual.